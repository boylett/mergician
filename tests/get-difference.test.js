const getDifference = require('../src/get-difference');

const arr1 = ['a', 'b', 'c'];
const arr2 = ['b', 'c', 'd'];
const arr3 = ['c', 'd', 'e'];

test('gets difference between two arrays', () => {
    const result = getDifference(arr1, arr2);

    expect(result).toMatchSnapshot();
});

test('gets difference between more than two arrays', () => {
    const result = getDifference(arr1, arr2, arr3);

    expect(result).toMatchSnapshot();
});
