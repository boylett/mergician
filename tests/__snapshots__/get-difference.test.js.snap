// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gets difference between more than two arrays 1`] = `
Array [
  "a",
  "e",
]
`;

exports[`gets difference between two arrays 1`] = `
Array [
  "a",
  "d",
]
`;
